docs/DesignDocs/Attestation_API_Proposal.md

```diff
@@ -159,15 +160,19 @@ generation and verification in
 manner.
 - Remove legacy artifacts that have become redundant, e.g. the SGX
 local / remote flags.
+- Incorporate changes proposed in [PR #3089](https://github.com/openenclave/openenclave/pull/3089).
 
 Proposed Headers files:
 
-- Attester public API to be declared in `openenclave/attestation/attester.h`
-- Verifier public API to be declared in `openenclave/attestation/verifier.h`
+- Attester public API to be declared in `<openenclave/attestation/attester.h>`
+- Verifier public API to be declared in `<openenclave/attestation/verifier.h>`
+- TEE-agnostic claim IDs to be declared in `<openenclave/bits/evidence.h>`
+- SGX-specific evidence format UUIDs and claim IDs to be declared in
+`<openenclave/attestation/sgx/evidence.h>`
 - Plugin API (helper functions and plugin entry points) to be declared in
-`openenclave/internal/plugin.h`
-- SGX evidence format UUIDs (and any other SGX-specific plugin related names)
-to be declared in `openenclave/internal/sgx/plugin.h`
+`<openenclave/internal/plugin.h>`
+- SGX-specific plugin related names to be declared in
+`<openenclave/internal/sgx/plugin.h>`
```

Comment:
Format UUIDs need to be declared in public header because public functions oe_get_evidence() and oe_verify_evidence() both need them. See similar issue fixed in PR 3131

Reply:
Yes, `<openenclave/attestation/sgx/evidence.h>` for those UUIDs is a public header.